pkg/systemd/logs.html

```diff
@@ -52,10 +52,14 @@
         </button>
         <ul id="prio-lists" class="dropdown-menu" role="menu">
           <li><a data-prio="*" translatable="yes">Everything</a></li>
-          <li><a data-prio="5" translatable="yes">Problems, Errors, Warnings, Notices</a></li>
-          <li><a data-prio="4" translatable="yes">Problems, Errors, Warnings</a></li>
-          <li><a data-prio="3" translatable="yes">Problems, Errors</a></li>
-          <li><a data-prio="2" translatable="yes">Only Problems</a></li>
+          <li><a data-prio="0" translatable="yes">Only Emergency</a></li>
```

Comment:
Wouldn't it be awesome to have this say "Emergency Only" instead? Then we would have all items starting with the logging level name :)